monkey/monkey_island/cc/setup.py

```diff
@@ -0,0 +1,24 @@
+from monkey_island.cc.services.attack.mitre_api_interface import MitreApiInterface
+from monkey_island.cc.models.attack_mitigation import AttackMitigation
+from monkey_island.cc.database import mongo
+from pymongo import errors
+
+
+def setup():
+    try_store_mitigations_on_mongo()
+
+
+def try_store_mitigations_on_mongo():
+    # import the 'errors' module from PyMongo
+    mitigation_collection_name = 'attack_mitigation'
+    try:
+        mongo.db.validate_collection(mitigation_collection_name)
+    except errors.OperationFailure:
+        mongo.db.create_collection(mitigation_collection_name)
+        store_mitigations_on_mongo()
```

Comment:
If the collection exists but is empty (or some of the mitigations are missing) this doesn't work - it doesn't fill the mitigations table.

To recreate this bug, empty the mitigations table and restart the Island. Somehow this happened to me organically,( I think because `store_mitigations_on_mongo` failed since I forgot to do `git submodule update`).

Reply:
Added a check for empty collection. Another solution could be like, don't manually delete data in collections, or something 🤷‍♂ :D